src/functions/sendCandidateResults/domain/template-id.model.ts

```diff
@@ -0,0 +1,20 @@
+export type CorrespondenceMethod = 'Email' | 'Letter';
+
+export enum Language {
+  CYMRAEG = 'Cymraeg',
```

Comment:
Would it be clearer to use English labels - just a thought :P

Reply:
I just made this enum to eliminate the magic strings, happy to change to WELSH if thats the right thing to do

Reply:
yeah I think that English label is clearer in this case